app/index.html

```diff
@@ -10,9 +10,9 @@
 
     <link rel="stylesheet" href="css/main.css">
   </head>
-  <body ng-app='theshelf'>
-    <side-content></side-content>
-    <div ui-view></div>
+  <body class='Site' ng-app='theshelf'>
```

Comment:
It's supposed to be the same.
I don't believe calling *Site* or *Layout* makes much difference now.
I'm just experimenting, so unless the name causes confusion, I'd say we could go with this.